### PR TITLE
Possibly use Requires instead of If block

### PR DESCRIPTION
**Pull request by webframp**
The check in the if block for security principle works, but there is a simpler way if you are already requiring PS 3.0 or higher. [Requires](https://technet.microsoft.com/en-us/library/hh847765.aspx) make this kind of check extremely easy.

No offense taken if you don't want to go this route.

### CODE REVIEW

**Comment by dougireton** [comment]
@webframp This looks great. Exactly the sort of code cleanup/refactoring this project needs. Thanks for your contribution. 

Expect this to be merged Monday.

File: bootstrap.ps1 [hunk 1]
```diff
@@ -12,11 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
-  [Security.Principal.WindowsBuiltInRole] "Administrator")) {
-    Write-Warning "You do not have Administrator rights to run this script!`nPlease re-run this script as an Administrator!"
-    Break
-}
+#Requires -Version 3.0
+#Requires -RunAsAdministrator
```
Comment (dougireton) [inline bootstrap.ps1]: @webframp unfortunately, `-RunAsAdministrator` is only supported in PowerShell 4.0 and above.

Can you resubmit your PR with this line removed? We are committed to supporting PowerShell 3.0 for now.

File: bootstrap.ps1 [hunk 2]
```diff
@@ -12,11 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
-  [Security.Principal.WindowsBuiltInRole] "Administrator")) {
-    Write-Warning "You do not have Administrator rights to run this script!`nPlease re-run this script as an Administrator!"
-    Break
-}
+#Requires -Version 3.0
```
Comment (dougireton) [inline bootstrap.ps1]: I'd like to keep this line.

**Comment by webframp** [comment]
Updated, per comments.

**Pull request merged**